express_cut/express_service/models.py

```diff
@@ -74,7 +74,7 @@ class Reservation(models.Model):
 class Feedback(models.Model):
     rating = models.SmallIntegerField(validators=[MinValueValidator(0), MaxValueValidator(5)])
     comments = models.CharField(max_length=200)
-    reservation = models.ForeignKey(Reservation, on_delete=models.CASCADE)
+    reservation = models.OneToOneField(Reservation, on_delete=models.CASCADE)
```

Comment:
Why the switch from Foreign to OneToOne? Is this something we should do on all other models as well?

Reply:
I did this because Feedback and Reservation is a one to one relationship, that is, you can only have one Feedback per Reservation and vice versa. If modeled as a foreign key it will not force uniqueness on the reservation attribute in the Feedback table.

Reply:
Ok makes sense. Think we need to check if there are any other models which need to be updated but gonna create an issue to keep track of it. Looks good!